bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -1089,77 +1095,104 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
+                            for (var i = 0; i < me.layerGeometries.geometry.getCoordinates().length; i++) {
+                                if(geometryType === "MultiPoint") {
+                                    var closestPoint = me.layerGeometries.geometry.getClosestPoint([me.clickCoords.x, me.clickCoords.y]);
+                                    var distance = (
+                                        Math.sqrt(
+                                            Math.pow((me.clickCoords.x - closestPoint[0]), 2) +
+                                            Math.pow((me.clickCoords.y - closestPoint[1]), 2)
+                                        ) / me.sandbox.getMap().resolution
+                                    );
+                                    if(distance <= 2.8) {
+                                        me.clickedGeometryNumber = i;
+                                        break;
+                                    }
+                                } else if(geometryType === "MultiPolygon") {
+                                    if(me.layerGeometries.geometry.intersectsCoordinate([me.clickCoords.x, me.clickCoords.y])) {
+                                        me.clickedGeometryNumber = i;
+                                        break;
+                                    }
+                                } else if(geometryType === "MultiLineString") {
+                                    if(exists) {
+                                        break;
+                                    }
+                                    var lineStrings = me.layerGeometries.geometry.getLineStrings();
+                                    //Have to transform point and linestring for turf.
+                                    var clickedPointCoordinatesWGS84 = ol.proj.transform([me.clickCoords.x,  me.clickCoords.y], me.sandbox.getMap()._projectionCode, 'EPSG:4326');
+                                    var pt = turf.point(clickedPointCoordinatesWGS84);
+                                    for(var j = 0; j < lineStrings.length; ++j) {
+                                        var lineStringCoords = lineStrings[j].getCoordinates();
+                                        var lineStringTransformed = [];
+                                        for(var k = 0; k < lineStringCoords.length; ++k) {
+                                            var lineStringPoint = lineStringCoords[k];
+                                            var lineStringPointWGS84 = ol.proj.transform(lineStringPoint, me.sandbox.getMap()._projectionCode, 'EPSG:4326');
+                                            lineStringTransformed.push(lineStringPointWGS84);
+                                        }
+                                        var line = turf.lineString(lineStringTransformed);
+                                        //Get default distance in kilometers
+                                        var distance = turf.pointToLineDistance(pt, line);
```

Comment:
For planar coordinate reference systems we could use `{'method': 'planar'}` as the third parameter (https://github.com/Turfjs/turf/tree/master/packages/turf-point-to-line-distance#pointtolinedistance). This way all the projection transformations could be avoided and the calculations pointToLineDistance makes would be more lightweight.